docs/architecture/README.md

```diff
@@ -27,60 +27,70 @@ If recorded decisions turned out to be lacking, convene a discussion, record the
-- [ADR-039-Peer-Behaviour](./adr-039-peer-behaviour.md)
-- [ADR-041-Proposer-Selection-via-ABCI](./adr-041-proposer-selection-via-abci.md)
-- [ADR-043-Blockchain-RiRi-Org](./adr-043-blockchain-riri-org.md)
-- [ADR-044-Lite-Client-With-Weak-Subjectivity](./adr-044-lite-client-with-weak-subjectivity.md)
-- [ADR-045-ABCI-Evidence](./adr-045-abci-evidence.md)
-- [ADR-046-Light-Client-Implementation](./adr-046-light-client-implementation.md)
-- [ADR-047-Handling-Evidence-From-Light-Client](./adr-047-handling-evidence-from-light-client.md)
-- [ADR-051-Double-Signing-Risk-Reduction](./adr-051-double-signing-risk-reduction.md)
-- [ADR-052-Tendermint-Mode](./adr-052-tendermint-mode.md)
-- [ADR-053-State-Sync-Prototype](./adr-053-state-sync-prototype.md)
-- [ADR-054-Crypto-Encoding-2](./adr-054-crypto-encoding-2.md)
-- [ADR-055-Protobuf-Design](./adr-055-protobuf-design.md)
-- [ADR-056-Light-Client-Amnesia-Attacks](./adr-056-light-client-amnesia-attacks)
-- [ADR-057-RPC](./adr-057-RPC.md)
-- [ADR-058-Event-Hashing](./adr-058-event-hashing.md)
-- [ADR-059-Evidence-Composition-and-Lifecycle](./adr-059-evidence-composition-and-lifecycle.md)
-- [ADR-060-Go-API-Stability](./adr-060-go-api-stability.md)
-- [ADR-061-P2P-Refactor-Scope](./adr-061-p2p-refactor-scope.md)
-- [ADR-062-P2P-Architecture](./adr-062-p2p-architecture.md)
-- [ADR-063-Privval-gRPC](./adr-063-privval-grpc.md)
+## Table of Contents
+
+### Accepted
+
+- [ADR-003: ABCI-APP-RPC](./adr-003-abci-app-rpc.md)
+- [ADR-004: Historical-Validators](./adr-004-historical-validators.md)
+- [ADR-006: Trust-Metric](./adr-006-trust-metric.md)
+- [ADR-009: ABCI-Design](./adr-009-ABCI-design.md)
+- [ADR-014: Secp-Malleability](./adr-014-secp-malleability.md)
+- [ADR-015: Crypto-Encoding](./adr-015-crypto-encoding.md)
+- [ADR-020: Block-Size](./adr-020-block-size.md)
+- [ADR-025: Commit](./adr-025-commit.md)
+- [ADR-034: Priv-Validator-File-Structure](./adr-034-priv-validator-file-structure.md)
+- [ADR-039: Peer-Behaviour](./adr-039-peer-behaviour.md)
+- [ADR-044: Lite-Client-With-Weak-Subjectivity](./adr-044-lite-client-with-weak-subjectivity.md)
+- [ADR-046: Light-Client-Implementation](./adr-046-light-client-implementation.md)
+- [ADR-047: Handling-Evidence-From-Light-Client](./adr-047-handling-evidence-from-light-client.md)
+- [ADR-053: State-Sync-Prototype](./adr-053-state-sync-prototype.md)
+- [ADR-055: Protobuf-Design](./adr-055-protobuf-design.md)
+- [ADR-056: Light-Client-Amnesia-Attacks](./adr-056-light-client-amnesia-attacks)
+- [ADR-059: Evidence-Composition-and-Lifecycle](./adr-059-evidence-composition-and-lifecycle.md)
+- [ADR-060: Go-API-Stability](./adr-060-go-api-stability.md)
+- [ADR-061: P2P-Refactor-Scope](./adr-061-p2p-refactor-scope.md)
+- [ADR-062: P2P-Architecture](./adr-062-p2p-architecture.md)
+
+### Rejected
+
+- [ADR-029: Check-Tx-Consensus](./adr-029-check-tx-consensus.md)
+- [ADR-058: Event-Hashing](./adr-058-event-hashing.md)
+
+
+### Proposed
```

Comment:
@marbar3778 I feel like some of these are already done/implemented and the ADRs just haven't been updated yet. Do you mind helping me groom through these to figure out which ones I can move to `Accepted` pls?

Reply:
Yea I can do this. Will open a separate pr

Reply:
Danke!